### DISCUSSION TITLE
error save map to image

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#518] [Q&A] [answered]
Hi all, is there any workaround to save or export map to image, since I use tool in map and there was error as below
![image](https://github.com/opengeos/leafmap/assets/141318843/f706e5f4-9257-4806-806f-856724d0c90b)

above error when I save to png

here error if save to jpg:
![image](https://github.com/opengeos/leafmap/assets/141318843/898d41ef-ed6a-4ee8-a727-1e9972296d72)

thanks

**Comment by giswqs** [comment, marked as answer]
It is essentially taking a screenshot. You are welcome to capture the screenshot manually if the code does not work.
